server/store/datastore/org.go

```diff
@@ -84,9 +81,8 @@ func (s storage) OrgFindByName(name string) (*model.Org, error) {
 
 func (s storage) orgFindByName(sess *xorm.Session, name string) (*model.Org, error) {
 	// sanitize
-	name = strings.ToLower(name)
 	org := new(model.Org)
-	return org, wrapGet(sess.Where("name = ?", name).Get(org))
+	return org, wrapGet(sess.Where("LOWER(name) = ?", strings.ToLower(name)).Get(org))
```

Comment:
This might be a mistake? I thought we don't want to sanitize calls/entries in the DB but only to the forge API.

Also removing sanitation in l87 and then re-adding it seems strange.

Reply:
The idea is to have searching case insensitive and storing case sensitive (the same way how forges do it, you can use both ways in urls etc and it shows the casing in the UI etc the user used).

> Also removing sanitation in l87 and then re-adding it seems strange.

It was previously missing the lower part on both sides. The change makes it unique with other calls.